docs/source/quantization-support.rst

```diff
@@ -440,7 +440,10 @@ Note that operator implementations currently only
 support per channel quantization for weights of the **conv** and **linear**
 operators. Furthermore the minimum and the maximum of the input data is
 mapped linearly to the minimum and the maximum of the quantized data
-type such that zero is represented with no quantization error.
+type such that zero is represented with no quantization error. This is slightly altered
+for symmetric qschemes, which instead use the negative and positive of the maximum absolute
+value of the input data in place of the normal minimum and maximum for the input data. For more info see how
+scale and zero point are calculated by :class:`~torch.ao.quantization.observer.MinMaxObserver`.
```

Comment:
this makes sense.  Thoughts about just writing out the formulas in addition to the text, to remove even more ambiguity?

Reply:
ya, good idea